docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -84,35 +76,103 @@ You can use any of the following repository types with searchable snapshots:
 
 [discrete]
 [[how-searchable-snapshots-work]]
 === How {search-snaps} work
 
 When an index is mounted from a snapshot, {es} allocates its shards to data
-nodes within the cluster. The data nodes then automatically restore the shard
-data from the repository onto local storage. Once the restore process
-completes, these shards respond to searches using the data held in local
-storage and do not need to access the repository. This avoids incurring the
-cost or performance penalty associated with reading data from the repository.
-
-If a node holding one of these shards fails, {es} automatically allocates it to
-another node, and that node restores the shard data from the repository. No
-replicas are needed, and no complicated monitoring or orchestration is
-necessary to restore lost shards.
-
-{es} restores {search-snap} shards in the background and you can search them
-even if they have not been fully restored. If a search hits a {search-snap}
-shard before it has been fully restored, {es} eagerly retrieves the data needed
-for the search. If a shard is freshly allocated to a node and still warming up,
-some searches will be slower. However, searches typically access a very small
-fraction of the total shard data so the performance penalty is typically small.
-
-Replicas of {search-snaps} shards are restored by copying data from the
-snapshot repository. In contrast, replicas of regular indices are restored by
+nodes within the cluster. The data nodes then automatically retrieve the
+relevant shard data from the repository onto local storage, based on the
+<<searchable-snapshot-mount-storage-options,mount options>> specified. If
+possible, searches use data from local storage. If the data is not available
+locally, {es} downloads the data that it needs from the snapshot repository.
+
+If a node holding one of these shards fails, {es} automatically allocates the
+affected shards on another node, and that node restores the relevant shard data
+from the repository. No replicas are needed, and no complicated monitoring or
+orchestration is necessary to restore lost shards. Although searchable snapshot
+indices have no replicas by default, you may add replicas to these indices by
+adjusting `index.number_of_replicas`. Replicas of {search-snap} shards are
+recovered by copying data from the snapshot repository, just like primaries of
+{search-snap} shards. In contrast, replicas of regular indices are restored by
 copying data from the primary.
 
+[discrete]
+[[searchable-snapshot-mount-storage-options]]
+==== Mount options
+
+To search a snapshot, you must first mount it locally as an index. Usually
+{ilm-init} will do this automatically, but you can also call the
+<<searchable-snapshots-api-mount-snapshot,mount snapshot>> API yourself. There
+are two possibilities for the `?storage` mount option, each with different
+performance characteristics and a different local storage footprint:
```

Comment:
Not sure if we should explicitly reference `?storage` since ILM will be the norm and there is no `?storage` option there.
```suggestion
are two options for mounting a snapshot, each with different performance
characteristics and local storage footprints:
```

Reply:
Yeah it's tricky, there are many options for mounting a snapshot, but we are only talking about this one parameter. That said, re-reading this after a break I prefer your wording.